casbin/util/builtin_operators.py

```diff
@@ -1,3 +1,4 @@
+import fnmatch
```

Comment:
plz add it to requirements.txt

Reply:
This module is part of the python standard library (like re or ipaddress). There's normally no need to add it to requirements.txt.